saleor/graphql/channel/types.py

```diff
@@ -121,6 +129,20 @@ class Meta:
         abstract = True
 
 
+class AllocationSettings(ObjectType):
+    allocation_strategy = AllocationStrategyEnum(
+        description=("Allocation strategy options."),
```

Comment:
I think we can drop brackets here:
```suggestion
        description="Allocation strategy options.",
```